src/utils/conversion.js

```diff
@@ -0,0 +1,99 @@
+export function convertAction(action, newProperties) {
+  const newAction = {
+    actionType: action.actionType,
+    bookmark: action.selectedBookmark,
+    // Will need to convert field selected by expression
+    field: action.selectedField,
+    // Need to convert expression for variable
+    variable: action.variable,
+    value: action.value,
+    softLock: action.softLock,
+    cId: action.cId,
+  };
+  switch (action.actionType) {
+    case 'clearOther':
+      newAction.actionType = 'clearAllButThis';
+      break;
+    case 'unlockAllAndClearAll':
+      newAction.actionType = 'clearAll';
+      newAction.softLock = true;
+      break;
+    case 'selectField':
+      newAction.actionType = 'selectValues';
+      break;
+    case 'selectAndLockField':
+      newProperties.actions.push({ ...newAction, cId: `${newAction.cId}1`, actionType: 'selectValues' });
```

Comment:
@haxxmaxx and @eliseeborn what do you think of the cId here? We make two actions from the old select and lock field action. However every action has a required cId. We could of course also try to create a new Id here, but then we would like to do that in the same way as has been done for the other Ids

Reply:
Hmm, I think they are randomly generated, but not unique. So we don't need to keep track of all the old ones to create new. But generating a cId would require putting that logic in the button repo I guess

Reply:
Yes and that is something I would like to avoid. However I say that cId is required, but have not tried saving without the cId. Let me try that actually

Reply:
OK, even if it's possible I think we should run it by Tobias

Reply:
So setting the cId to null seems to work. This ID is used to have the order in place for the items so I looked at what happens when you change the order. In that case a new Id is generated for the ones that don't have one yet. So my suggestion would be to set to null. @Caele what do you think?

Reply:
So the ordering of the items is based on random generated cIds? We probably need to do something about the order anyway

Reply:
Did we still need the cId?

Reply:
We don't need it, but it is automatically generated when you use an array in the property panel

Reply:
ok, then it should not matter